utils/generate-bags/node-runtime/src/main.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024, Horizen Labs, Inc.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Make the set of bag thresholds to be used with pallet-bags-list.
+
+use clap::Parser;
+use generate_bags::generate_thresholds;
+use std::path::PathBuf;
+use subxt::{OnlineClient, SubstrateConfig};
+
+#[derive(Debug, Parser)]
+// #[clap(author, version, about)]
+struct Opt {
+    /// How many bags to generate.
+    #[arg(long, default_value_t = 200)]
+    n_bags: usize,
```

Comment:
Let's probably agree with John how to set this value

Reply:
asked John, waiting on reply

Reply:
200 is fine for now, asked John